### DISCUSSION TITLE
p/958e1a550f804fb8b755040bbd040bd2/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#58] [Announcements]
# p/958e1a550f804fb8b755040bbd040bd2/

背景Poe 是一个 AI 聊天机器人，它支持多种 AI 模型，包括 GPT-4o、Claude 3.5 Sonnet、Gemini Pro 等。还支持各种类型的 Bot，其中 Server Bot 是最自由的，可以自己编写 Bot 的逻辑。但是，Poe 的 Server Bot 官方仅支持 Python，而吾辈更喜欢 JavaScript，所以研究了一下怎么实现。 初始化项目一开始使用 expre

https://blog.rxliuli.com/p/958e1a550f804fb8b755040bbd040bd2/

**Comment by lianlilin** [comment]
偶然搜到了一个你的关于fs-extra对esm支持度的文档，翻了下你的博客，真的是坚持写作多年那，而且在持续学习~佩服，respect

**Reply by rxliuli** [reply]
你也可以开始写呀，开发者可能是这个星球上最喜欢分享的群体了🤪
